starfish/munge.py

```diff
@@ -30,12 +32,64 @@ def scale(stack, metric, clip=False):
     return list_to_stack(res)
 
 
-def gather(df, key, value, cols):
-    id_vars = [col for col in df.columns if col not in cols]
-    id_values = cols
-    var_name = key
-    value_name = value
-    return pd.melt(df, id_vars, id_values, var_name, value_name)
+def melt(df: pd.DataFrame, new_index_name: Any, new_value_name: Any, melt_columns: Iterable) -> pd.DataFrame:
```

Comment:
melt_columns is not Iterable[str]?

(i actually don't know, and am too lazy to check)